src/main/java/BaseBallGame.java

```diff
@@ -0,0 +1,107 @@
+public class BaseBallGame {
+    final int chance;
+    final String randomNum;
```

Comment:
최대한 변수명에는 약자는 사용하지 않는게 좋습니다.

```suggestion
    final String randomNumber;
```